main.tf

```diff
@@ -4,8 +4,118 @@ provider "aws" {
   region     = var.region
 }
 
-resource "aws_instance" "example" {
-  ami           = "ami-00bf61217e296b409"
-  instance_type = "t2.micro"
-  key_name      = var.key_name
+resource "aws_ecs_cluster" "riht" {
+  name = var.name
```

Comment:
What's does `name` field dictate? The name of the cluster? If so, what's `"riht"`?

Reply:
Resource names like `"riht"` are exclusively used just within Terraform and don't show up in AWS. You just use them to reference other Terraform resources, so it's kinda like a variable name in a programming language.

The `name` field is the actual name of the cluster in Terraform, yeah.